test/nightwatch/table/table-test.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  '@tags': ['custom', 'table'],
+  'disabled': true,
```

Comment:
just curious - are the table tests disabled because they are a work in progress?